spec/store/index-spec.js

```diff
@@ -293,17 +294,16 @@ describe("Store", () => {
   describe("dispose", () => {
     it("should dispose kernels and subscriptions", () => {
       spyOn(store.subscriptions, "dispose");
-      spyOn(store.markers, "clear");
       const kernel1 = jasmine.createSpyObj("kernel1", ["destroy"]);
       const kernel2 = jasmine.createSpyObj("kernel2", ["destroy"]);
       store.runningKernels = [kernel1, kernel2];
       store.dispose();
       expect(store.runningKernels.length).toEqual(0);
+      expect(store.markersMapping.size).toEqual(0);
       expect(store.kernelMapping.size).toBe(0);
       expect(kernel1.destroy).toHaveBeenCalled();
       expect(kernel2.destroy).toHaveBeenCalled();
       expect(store.subscriptions.dispose).toHaveBeenCalled();
-      expect(store.markers.clear).toHaveBeenCalled();
```

Comment:
@BenRussert 	I couldn't figure out a way to `expect` the `clear` on the values of the `Map`...

Reply:
I think you would just want to change this to `store.markersMapping.clear`, and then above to ` spyOn(store.markersMapping, "clear");`